.gitignore

```diff
@@ -11,6 +11,9 @@
 /db/*.sqlite3
 /db/*.sqlite3-*
 
+# Ignore OS-specefic hidden files.
+.DS_Store
```

Comment:
This is good to have here!  I'd also recommend adding a local environment config file like `~/.gitignore_global` with this content:

```
*~
\#*\#
.DS_Store
```

And then you can run this command to use that file as a global ignore list:

`git config --global core.excludesFile '~/.gitignore_global'`

That way, if you're ever working with a repository where someone else forgot to add .DS_Store to the repo's .gitignore, you can avoid accidentally committing .DS_Store (and other globally ignored files) to it.